src/styles/variables.scss

```diff
@@ -1,11 +1,10 @@
-$background-primary: #1a1a1a;
-$background-secondary: #2b2b2b;
-$background-tertiary: #3c3c3c;
-$text-primary: #d9d9d9;
-$text-secondary: #d9d9d9b3;
-$accent-color-red: #fb2667;
-$accent-color-blue: #48c7d9;
-$border-color: #d9d9d933;
+$background-primary: #111319;
+$background-secondary: #13161D;
+$text-primary: #96DBF9;
+$text-secondary: #39B8FF;
+$text-tertiary: #3E7C97;
+$accent-color-blue: #39B8FF;
```

Comment:
Since you removed the variable `accent-color-red`, you will have to go through all the other `.scss` files and replace it with the new `accent-color-blue` variable.